editor/editor_settings.cpp

```diff
@@ -246,8 +246,8 @@ static void _create_script_templates(const String &p_path) {
 	dir->change_dir(p_path);
 	for (int i = 0; i < keys.size(); i++) {
 		if (!dir->file_exists(keys[i])) {
-			file->reopen(p_path.plus_file((String)keys[i]), FileAccess::WRITE);
-			ERR_FAIL_COND(!file);
+			Error err = file->reopen(p_path.plus_file((String)keys[i]), FileAccess::WRITE);
+			ERR_FAIL_COND(!err);
```

Comment:
I think you should use `err != OK` (or `Error::OK`, if needed). This way, the error message would be clearer.